galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/beans/SubmitRunRequest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.beans;
```

Comment:
Also just wondering why we don't use a generated bean ?

Reply:
This is a bean internal to the framework rather than the API server. There are some small differences between the JSON payload sent to the API server to submit tests and the structure used internally within the framework (e.g. one includes a list of test classes to submit, the other represents a single test), hence the separation.